prometheus-high-performance.yaml

```diff
@@ -0,0 +1,304 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: prometheus
+  annotations:
+    "openshift.io/display-name": Prometheus
+    description: |
+      A monitoring solution for an OpenShift cluster - collect and gather metrics from nodes, services, and the infrastructure. This is a tech preview feature.
+    iconClass: icon-cogs
+    tags: "monitoring,prometheus,time-series"
+parameters:
+- description: The namespace to instantiate prometheus under. Defaults to 'kube-system'.
+  name: NAMESPACE
+  value: kube-system
+- description: The location of the proxy image
+  name: IMAGE_PROXY
+  value: openshift/oauth-proxy:v1.0.0
+- description: The location of the prometheus image
+  name: IMAGE_PROMETHEUS
+  value: openshift/prometheus:v2.0.0-dev
+- description: The session secret for the proxy
+  name: SESSION_SECRET
+  generate: expression
+  from: "[a-zA-Z0-9]{43}"
+- description: Time interval for scrape config (1m, 5s)
+  name: SCRAPE_INTERVAL
+  value: 15s
+objects:
+# Authorize the prometheus service account to read data about the cluster
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: prometheus
+    namespace: "${NAMESPACE}"
```

Comment:
Where do these shell variables get resolved? This is an implementation detail I do not yet understand.

Reply:
currently it is hard-coded based on the original origin code 
https://github.com/mbruzek/ansible-openstack-setup/blob/9a786bd5ffb61c1f880b72730ed05586d44230d0/prometheus-high-performance.yaml#L14

Reply:
I see it now thank you!